ISSUE_TEMPLATE/new_meetup.md

```diff
@@ -0,0 +1,26 @@
+***Please help us by filling out the text below. All answers are optional***
+
+# New Meetup
+
+## Where
+
+*A short description of the event. Please also add any links, e.g. to the meetup page.*
+
+## When
+
+*the date. please name the timezone*
+
+[ ] I'd like the event to be public calendar
+[ ] It's a repeat event, I would like to have access to the calender to do things on my own
+
+## Support
+
+[ ] I need coaching
+[ ] I need contacts in my area
+[ ] I need help promoting
+[ ] I'd like to have stickers
+[ ] I'd like to have swag and stickers
```

Comment:
To be honest even as an experienced organizer I'd tick all these boxes because it's always nice to have. If we really want to offer all these services we need some upfront investment by the organizer (something as simple as providing their address for stickers).

So, for swag as a concrete example, how about:

```
If you're interested in obtaining stickers and/or swag, please send a short email to community-swag@rust-lang.org, providing

- Your physical mail address and phone number
- The projected number of attendees for your event
- How many events you're planning to run in the future
- Optionally, if you're interested in a certain type of swag for specific reasons (e.g. you're running a T-Shirt give away)
```